src/ducks/components/PushBanners/helpers.js

```diff
@@ -0,0 +1,38 @@
+import BannerForAA from './BannerForAA'
+import BannerForPass from './BannerForPass'
+
+export const makePushBanner = (oAuthClients, setting) => {
+  const { hasAAClient, hasPassClient } = oAuthClients.reduce(
```

Comment:
nit: Using 2 `array.some()` would be easier to read. It is maybe less optimized but we shouldn't have tons of oauth clients, so it wouldn't matter.

```js
const hasAAClient = oAuthClients.some(client => software_id === 'amiral') 
const hasPassClient = oAuthClients.some(client => software_id === 'io.cozy.pass.mobile') 
```

Reply:
ok si tu veux